<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultWindow</class>
 <widget class="QMainWindow" name="ResultWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="2">
       <widget class="QGroupBox" name="groupBoxFilter">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox {
    border: 1px solid #ddd;
    border-radius: 5px;
    margin-top: 10px;
    font-size: 16px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
    background-color: #F0F0F0;
}

QGroupBox::indicator {
    width: 18px;
    height: 18px;
    border: 1px solid #ddd;
    border-radius: 9px;
}

QGroupBox::indicator:checked {
    background-color: #2196f3;
    border-color: #2196f3;
}
</string>
        </property>
        <property name="title">
         <string>发布</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QComboBox" name="comboEventsPublish">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="BtnF">
           <property name="text">
            <string>发布决赛</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>项目</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="BtnR">
           <property name="text">
            <string>发布复赛</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true">QGroupBox {
    border: 1px solid #ddd;
    border-radius: 5px;
    margin-top: 10px;
    font-size: 16px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
    background-color: #F0F0F0;
}

QGroupBox::indicator {
    width: 18px;
    height: 18px;
    border: 1px solid #ddd;
    border-radius: 9px;
}

QGroupBox::indicator:checked {
    background-color: #2196f3;
    border-color: #2196f3;
}
</string>
        </property>
        <property name="title">
         <string>过滤</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QRadioButton" name="rBtnR">
           <property name="styleSheet">
            <string notr="true">QRadioButton {
    font-size: 14px;
}

QRadioButton::indicator {
    width: 8px;
    height: 8px;
    border-radius: 50%;
    border: 2px solid #C0C0C0;
}

QRadioButton::indicator:checked {
    border-color: #0078D7;
    background-color: #0078D7;
}

QRadioButton::indicator:hover {
    border-color: #0078D7;
}

QRadioButton::indicator:checked:hover {
    background-color: #0078D7;
}
</string>
           </property>
           <property name="text">
            <string>复赛</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="rBtnP">
           <property name="styleSheet">
            <string notr="true">QRadioButton {
    font-size: 14px;
}

QRadioButton::indicator {
    width: 8px;
    height: 8px;
    border-radius: 50%;
    border: 2px solid #C0C0C0;
}

QRadioButton::indicator:checked {
    border-color: #0078D7;
    background-color: #0078D7;
}

QRadioButton::indicator:hover {
    border-color: #0078D7;
}

QRadioButton::indicator:checked:hover {
    background-color: #0078D7;
}
</string>
           </property>
           <property name="text">
            <string>初赛</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="rBtnF">
           <property name="styleSheet">
            <string notr="true">QRadioButton {
    font-size: 14px;
}

QRadioButton::indicator {
    width: 8px;
    height: 8px;
    border-radius: 50%;
    border: 2px solid #C0C0C0;
}

QRadioButton::indicator:checked {
    border-color: #0078D7;
    background-color: #0078D7;
}

QRadioButton::indicator:hover {
    border-color: #0078D7;
}

QRadioButton::indicator:checked:hover {
    background-color: #0078D7;
}
</string>
           </property>
           <property name="text">
            <string>决赛</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QRadioButton" name="rBtnAll">
           <property name="styleSheet">
            <string notr="true">QRadioButton {
    font-size: 14px;
}

QRadioButton::indicator {
    width: 8px;
    height: 8px;
    border-radius: 50%;
    border: 2px solid #C0C0C0;
}

QRadioButton::indicator:checked {
    border-color: #0078D7;
    background-color: #0078D7;
}

QRadioButton::indicator:hover {
    border-color: #0078D7;
}

QRadioButton::indicator:checked:hover {
    background-color: #0078D7;
}
</string>
           </property>
           <property name="text">
            <string>所有赛程</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QGroupBox" name="groupBoxSearch">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox {
    border: 1px solid #ddd;
    border-radius: 5px;
    margin-top: 10px;
    font-size: 16px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
    background-color: #F0F0F0;
}

QGroupBox::indicator {
    width: 18px;
    height: 18px;
    border: 1px solid #ddd;
    border-radius: 9px;
}

QGroupBox::indicator:checked {
    background-color: #2196f3;
    border-color: #2196f3;
}
</string>
        </property>
        <property name="title">
         <string>检索</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="3">
          <widget class="QPushButton" name="btnSearch">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #3385ff;
    border: none;
    color: #fff;
    padding: 3px 10px;
    border-radius: 3px;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #2d78f4;
}
</string>
           </property>
           <property name="text">
            <string>学号搜索</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="editStudentID"/>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboEventsID">
           <property name="styleSheet">
            <string notr="true">min-width: 3em;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBoxSort">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox {
    border: 1px solid #ddd;
    border-radius: 5px;
    margin-top: 10px;
    font-size: 16px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
    background-color: #F0F0F0;
}

QGroupBox::indicator {
    width: 18px;
    height: 18px;
    border: 1px solid #ddd;
    border-radius: 9px;
}

QGroupBox::indicator:checked {
    background-color: #2196f3;
    border-color: #2196f3;
}
</string>
        </property>
        <property name="title">
         <string>排名</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1">
          <widget class="QComboBox" name="comboStatus">
           <property name="styleSheet">
            <string notr="true">min-width: 3em;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>赛程</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>项目</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="comboEventsRank">
           <property name="styleSheet">
            <string notr="true">min-width: 3em;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QPushButton" name="BtnRank">
           <property name="text">
            <string>确认排名</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>743</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actEditResult"/>
   <addaction name="actDeleteResult"/>
   <addaction name="actCheckNewRecord"/>
   <addaction name="actExportResult"/>
   <addaction name="separator"/>
   <addaction name="actClose"/>
  </widget>
  <action name="actEditResult">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/images/124.bmp</normaloff>:/icons/images/124.bmp</iconset>
   </property>
   <property name="text">
    <string>编辑成绩</string>
   </property>
   <property name="toolTip">
    <string>编辑成绩</string>
   </property>
  </action>
  <action name="actExportResult">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/images/Excel.ico</normaloff>:/icons/images/Excel.ico</iconset>
   </property>
   <property name="text">
    <string>导出信息</string>
   </property>
   <property name="toolTip">
    <string>导出成绩信息</string>
   </property>
  </action>
  <action name="actClose">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/images/706.bmp</normaloff>:/icons/images/706.bmp</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭</string>
   </property>
  </action>
  <action name="actCheckNewRecord">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/images/103.bmp</normaloff>:/icons/images/103.bmp</iconset>
   </property>
   <property name="text">
    <string>确认破纪录</string>
   </property>
   <property name="toolTip">
    <string>确认破纪录</string>
   </property>
  </action>
  <action name="actDeleteResult">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/images/324.bmp</normaloff>:/icons/images/324.bmp</iconset>
   </property>
   <property name="text">
    <string>删除成绩</string>
   </property>
   <property name="toolTip">
    <string>删除成绩</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actClose</sender>
   <signal>triggered()</signal>
   <receiver>ResultWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
